tools/fix_timepoint_date_problems.php

```diff
@@ -483,7 +483,7 @@ function diagnose($sessionID, $dateType=null, $newDate=null)
             fwrite(STDERR, "Age at $stage: $age [ $dateBirth $dateOfStage]\n");
 
             // create battery object
-            $battery =& new NDB_BVL_Battery();
+            $battery =& NDB_BVL_Battery();

```

Comment:
Yeah I'm sure it doesn't produce errors or anything. You just made a change on line 254 that seems to update the same line of code and it's inconsistent with this one.